schema.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS user;
+
+CREATE TABLE user(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name TEXT UNIQUE NOT NULL,
+    password TEXT NOT NULL
+);
```

Comment:
Are you sure you still need this?

Reply:
No, deleted it. https://github.com/mstoro/nasa_api/pull/1/commits/fc29d10da877dcf14722e0f9111d975be3acde1b